### PR TITLE
[nvidia-bluefield] Add config inclusions for Nvidia Bluefield-3 DPU.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

File: patch/kconfig-secure-boot-inclusions [hunk 1]
```diff
@@ -11,7 +11,7 @@ CONFIG_MODULE_SIG_FORCE=y
 CONFIG_SYSTEM_TRUSTED_KEYS="debian/certs/debian-uefi-certs.pem"
 CONFIG_MODULE_SIG_HASH="sha512"
 CONFIG_MODULE_SIG_SHA512=y
-CONFIG_KEXEC_SIG_FORCE=y
```
Comment (vivekkarri) [inline patch/kconfig-secure-boot-inclusions:14]: Why is this a problem for arm64 and not amd64?

Comment (oleksandrivantsiv) [reply patch/kconfig-secure-boot-inclusions:14]: Due to the difference in the architecture and set of the drivers

Comment (oleksandrivantsiv) [reply patch/kconfig-secure-boot-inclusions:14]: CONFIG_KEXEC_SIG_FORCE is not available for arm64. It is basically copy-paste that no one tested

Comment (vivekkarri) [reply patch/kconfig-secure-boot-inclusions:14]: I see. Approved

**Review by vivekrnv — APPROVED** [review]

**Pull request closed without merging**